tests/h/schemas/admin_group_test.py

```diff
@@ -79,76 +79,66 @@ def test_it_raises_if_no_origins(self, group_data, bound_schema):
         with pytest.raises(colander.Invalid, match='At least one origin'):
             bound_schema.deserialize(group_data)
 
-    def test_it_raises_if_group_type_changed(self, group_data, pyramid_csrf_request):
+    def test_it_raises_if_group_type_changed(self, group_data,
+                                             pyramid_csrf_request, org, user_svc):
         group = mock.Mock(type='open')
         group_data['group_type'] = 'restricted'
-        schema = CreateAdminGroupSchema().bind(request=pyramid_csrf_request, group=group)
+        schema = CreateAdminGroupSchema().bind(request=pyramid_csrf_request,
+                                               group=group,
+                                               user_svc=user_svc,
+                                               organizations=[org])
 
         with pytest.raises(colander.Invalid, match='Changing group type'):
             schema.deserialize(group_data)
 
-    def test_it_does_not_raise_if_group_type_is_same(self, group_data, pyramid_csrf_request):
+    def test_it_does_not_raise_if_group_type_is_same(self, group_data,
+                                                     pyramid_csrf_request, org, user_svc):
         group = mock.Mock(type='open')
         group_data['group_type'] = 'open'
-        schema = CreateAdminGroupSchema().bind(request=pyramid_csrf_request, group=group)
+        schema = CreateAdminGroupSchema().bind(request=pyramid_csrf_request,
+                                               group=group,
+                                               user_svc=user_svc,
+                                               organizations=[org])
 
         schema.deserialize(group_data)
 
-    def test_it_raises_if_member_invalid(self, group_data, pyramid_csrf_request, user_svc):
+    def test_it_raises_if_member_invalid(self, group_data, bound_schema, user_svc):
         user_svc.fetch.return_value = None
-        bound_schema = CreateAdminGroupSchema().bind(request=pyramid_csrf_request, authority='some_authority')
-
         group_data['members'] = ["user_who_does_not_exist"]
         with pytest.raises(colander.Invalid, match='members.*Username not found'):
             bound_schema.deserialize(group_data)
 
-    @pytest.fixture
-    def user_svc(self, pyramid_config):
-        svc = mock.create_autospec(UserService, spec_set=True, instance=True)
-        pyramid_config.register_service(svc, name='user')
-        return svc
-
-
-class TestCreateSchemaWithValidator(object):
-
-    def test_it_passes_creator_and_authority_to_service(self,
+    def test_it_allows_when_creator_exists_at_authority(self,
```

Comment:
Is there a test for when the user's authority doesn't match the orgs authority? I don't see one here but I could just be missing it. I find these test names difficult to follow. Seems like we could use some refactoring here to be in a more readable format. Ex: test_validation_passes_when_creator_belongs_to_authority. (And I also realize they were like this before this PR so it's up to you.)